### PR TITLE
Release 0.18.0

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/build_mamba.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		-DCMAKE_INSTALL_PREFIX=$PREFIX 	\
+		-DPython_EXECUTABLE=$PYTHON     \
+		-DBUILD_LIBMAMBAPY=ON
+fi
+
+ninja
+
+if [[ $PKG_NAME == "libmamba" ]]; then
+    strip libmamba/libmamba.so
+	strip mamba_package/mamba-package
```
Comment (adriendelsalle) [inline recipe/build_mamba.sh:37]: I would like to strip `mamba-package` executable but it looks like it doesn't work as expected

**Review by adriendelsalle — COMMENTED** [review]
the `pypy` build of `libmambapy` also doesn't have the `pypy` build string and I don't know what should be changed to make it detected and automatically hashed correctly

File: recipe/build_mamba.sh (hunk 1 above)
Comment (adriendelsalle) [reply recipe/build_mamba.sh:37]: we should also have `strip libmamba/libmamba.so*`

**Comment by wolfv** [comment]
This has to go under `build->string`, e.g. https://github.com/conda-forge/tensorflow-feedstock/blob/31f96910cdb8433733c4ba18b55aa592777a556d/recipe/meta.yaml#L100-L106

File: recipe/build_mamba.bat [hunk 2]
```diff
@@ -0,0 +1,42 @@
+if (%PKG_NAME% == "mamba") (
+	cd mamba
+	%PYTHON% -m pip install . --no-deps -vv
+	if (exist %PREFIX%\condabin) (
+		echo "Adding link to mamba into condabin";
+		mklink %PREFIX%/condabin/mamba %LIBRARY_PREFIX%/bin/mamba
```
Comment (wolfv) [inline recipe/build_mamba.bat:6]: did you compare that to the previous recipe? I think we only need that on Unix :)

File: recipe/meta.yaml [hunk 3]
```diff
@@ -41,6 +41,12 @@ outputs:
         - termcolor-cpp
         - cli11
         - libiconv                               # [unix]
+    test:
+      commands:
+        - test -e $PREFIX/lib/libmamba.so  # [linux]
```
Comment (wolfv) [inline recipe/meta.yaml:46]: you can use `${SHLIB_EXT}`, e.g. https://github.com/conda-forge/nss-feedstock/blob/0b568c7e2d505eb8f1c3f55f06328588657318c7/recipe/meta.yaml#L41-L47

Comment (adriendelsalle) [reply recipe/meta.yaml]: done!

**Comment by bollwyvl** [comment]
Hooray!

**Pull request merged**